WebServers/HTTPServer/HttpProcessor.cs

```diff
@@ -318,8 +325,31 @@ public void HandleClient(TcpClient? tcpClient, ushort ListenerPort)
                                     {
                                         default:
 
+                                            #region Dust 514 dcrest
+                                            //CCPGamesDomains.Contains(Host) || 
+                                            if (Host.Contains("26004") //Check for Dust514 specific Port!!
```

Comment:
I would like a more precise fetching, use the CCPGames domain you delcare earlier.

Reply:
I cannot as the game doesn't send the domain as the host

Reply:
So I must use port as next reliable method to fetch, and 26004 is Dust 514 listen port for the DCrest API

Reply:
Ah ok! You can use the serverPort attribute afaik for that.